src/lib.rs

```diff
@@ -181,61 +200,6 @@ mod aes128gcm_tests {
         assert_eq!(ciphertext, "0c6bfaadad67958803092d454676f397000010004104fe33f4ab0dea71914db55823f73b54948f41306d920732dbb9a59a53286482200e597a7b7bc260ba1c227998580992e93973002f3012a28ae8f06bbb78e5ec0ff297de5b429bba7153d3a4ae0caa091fd425f3b4b5414add8ab37a19c1bbb05cf5cb5b2a2e0562d558635641ec52812c6c8ff42e95ccb86be7cd");
     }
 
-    #[test]
```

Comment:
We had quite a few tests for for dealing with record boundaries etc. Per the discussion in https://github.com/mozilla/application-services/issues/1068, we do not need this for webpush, and I don't think it works right in the legacy `aesgcm` scheme anyway.

I intend to push a follow-up PR that removes the multi-record stuff entirely; it seemed like too much churn to do all at once in this PR.